webvtt.html

```diff
@@ -3688,6 +3688,12 @@
   <section>
    <h2>Rendering</h2>
 
+   <p class="note">This section contains an algorithm that describes in some detail how to visually
```

Comment:
I think we can drop "contains an algorithm that" and "the content of" to make this a bit tighter.